core/types/validation/keywords.go

```diff
@@ -0,0 +1,837 @@
+package validation
+
+import "strings"
+
+func IsKeyword(str string) bool {
+	upperWord := strings.ToUpper(str)
+
+	_, ok := reservedWords[upperWord]
+	return ok
+}
+
+var reservedWords = map[string]struct{}{
```

Comment:
Were these files supposed to be a `git mv` from `common/validation`?

Reply:
Yes